Makefile

```diff
@@ -10,10 +10,7 @@ MANINSTALLDIR=${PREFIX}/share/man
 GO_MD2MAN ?= /usr/bin/go-md2man
 
 ifeq ($(DEBUG), 1)
-  GOGCFLAGS += "-N -l"
-else
-  GOGCFLAGS ?= ""
-  DEBUG ?= 0
+  override GOGCFLAGS += -N -l
```

Comment:
If you set `GOGCFLAGS=` on the command line, it completely overrides what `Makefile` does and the `GOGCFLAGS += …` line does nothing, making `DEBUG=1` ineffective.